ci-operator/config/openshift/release/openshift-release-master__nightly-4.12.yaml

```diff
@@ -152,6 +156,32 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-single-node-workers-upgrade-conformance
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported
+  cron: 22 18 7,14,23,30 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:oldest-supported-z-rollback
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_LATEST},${RELEASE_IMAGE_Z_ROLLBACK}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_Z_ROLLBACK
+        name: release:oldest-supported-z-rollback
+      - env: RELEASE_IMAGE_LATEST
+        name: release:latest
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - ref: cucushift-chainupgrade-toimage
+    - chain: openshift-upgrade-qe-sanity
```

Comment:
Shall we drop this chain, and move the last hop from 4.12.z-4.12.0 into above `cucushift-chainupgrade-toimage`, then it's a two-hop upgrade.

Reply:
I'm wondering how we do the change, by updating the `cucushift-chainupgrade-toimage` step or updating the update edge? Generally `cucushift-chainupgrade-toimage` is used to do the chainupgrade to the 2nd last image. I'd like to make rollback test consistent with chainupgrade.

Reply:
> Generally cucushift-chainupgrade-toimage is used to do the chainupgrade to the 2nd last image. I'd like to make rollback test consistent with chainupgrade.

I agree with this point.

Reply:
> updating the update edge

This sounds much simpler. for example, move 4.12.z,4.12.0,4.12.dummy to the update edge, 4.12.dummy will be dropped, so actually only two hops.

> Generally cucushift-chainupgrade-toimage is used to do the chainupgrade to the 2nd last image. I'd like to make rollback test consistent with chainupgrade.

This is because they are for different purpose, right? In the last hop, we have to do pre/post upgrade test. But in rollback test, there is not pre/post upgrade test, so i think we could move it together to use the chainupgrade to the best advantage and drop dup healthy check.

Reply:
Per sync result, we are agreed to get it updated in another pr.